grails-app/views/stockMovement/_comments.gsp

```diff
@@ -12,7 +12,7 @@
             </tr>
             </thead>
             <tbody>
-            <g:each var="comment" in="${requisition?.comments}">
+            <g:each var="comment" in="${requisition?.comments?.sort(){ a,b -> b?.lastUpdated <=> a?.lastUpdated }}">
```

Comment:
Maybe we can create a method for comparing `lastUpdated` dates in the domain, and use the `.sort()` method without passing closure?

Reply:
Just like Kacper suggested, I'll overwrite compareTo and call sort() in the GSP.